ipn/localapi/cert.go

```diff
@@ -185,7 +186,10 @@ func (h *Handler) getCertPEM(ctx context.Context, logf logger.Logf, traceACME fu
 	if err != nil {
 		return nil, fmt.Errorf("acmeKey: %w", err)
 	}
-	ac := &acme.Client{Key: key}
+	ac := &acme.Client{
+		Key:       key,
+		UserAgent: "tailscaled " + version.Long,
```

Comment:
don't you want a slash instead of a space? https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent

Reply:
A slash would be consistent with the standard use of the User Agent header and other ACME clients, though from Let's Encrypt's perspective we don't care too much about the separator.

Reply:
Good call; switched to a slash.